commit fc99627fe1f109d6e25897cdb2f7148fe8c1737f (refs/changes/86/5086/2)
Author: Tim Foster <tim.foster@joyent.com>
Date:   2018-11-21T17:08:33+00:00 (11 months ago)
    
    MANTA-4022 propeller's nodunit dep no longer installable with prebuilt node version

diff --git a/Makefile b/Makefile
index ab8b4f0..2e659ef 100644
--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@
 #
 
 #
-# Copyright (c) 2014, Joyent, Inc.
+# Copyright (c) 2018, Joyent, Inc.
 #
 
 #
@@ -81,11 +81,6 @@ $(NODEUNIT): | $(NPM_EXEC)
 
 CLEAN_FILES += $(NODEUNIT) ./node_modules/nodeunit
 
-.PHONY: test
-test: $(NODEUNIT)
-	mkdir -p ./tmp
-	find test/ -name '*.test.js' | xargs -n 1 $(NODEUNIT)
-
 .PHONY: scripts
 scripts: deps/manta-scripts/.git
 	mkdir -p $(BUILD)/scripts
diff --git a/package.json b/package.json
index 5734196..3f7858a 100644
--- a/package.json
+++ b/package.json
@@ -15,9 +15,6 @@
                 "manta": "1.4.3",
                 "vasync": "1.5.0"
         },
-        "devDependencies": {
-                "nodeunit": "0.7.4"
-        },
         "engines": {
                 "node": "0.10.30"
         },
diff --git a/test/helper.js b/test/helper.js
deleted file mode 100644
index 19771e8..0000000
--- a/test/helper.js
+++ /dev/null
@@ -1,75 +0,0 @@
-/*
- * This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this
- * file, You can obtain one at http://mozilla.org/MPL/2.0/.
- */
-
-/*
- * Copyright (c) 2014, Joyent, Inc.
- */
-
-var bunyan = require('bunyan');
-
-
-
-///--- Helpers
-
-function createLogger(name, stream) {
-        var log = bunyan.createLogger({
-                level: (process.env.LOG_LEVEL || 'info'),
-                name: name || process.argv[1],
-                stream: stream || process.stdout,
-                src: true,
-                serializers: bunyan.stdSerializers
-        });
-        return (log);
-}
-
-
-
-///--- Exports
-
-module.exports = {
-
-        after: function after(teardown) {
-                module.parent.exports.tearDown = function _teardown(callback) {
-                        try {
-                                teardown.call(this, callback);
-                        } catch (e) {
-                                console.error('after:\n' + e.stack);
-                                process.exit(1);
-                        }
-                };
-        },
-
-        before: function before(setup) {
-                module.parent.exports.setUp = function _setup(callback) {
-                        try {
-                                setup.call(this, callback);
-                        } catch (e) {
-                                console.error('before:\n' + e.stack);
-                                process.exit(1);
-                        }
-                };
-        },
-
-        test: function test(name, tester) {
-                module.parent.exports[name] = function _(t) {
-                        var _done = false;
-                        t.end = function end() {
-                                if (!_done) {
-                                        _done = true;
-                                        t.done();
-                                }
-                        };
-                        t.notOk = function notOk(ok, message) {
-                                return (t.ok(!ok, message));
-                        };
-
-                        tester.call(this, t);
-                };
-        },
-
-        createLogger: createLogger
-
-};
diff --git a/test/stub.test.js b/test/stub.test.js
deleted file mode 100644
index b77f392..0000000
--- a/test/stub.test.js
+++ /dev/null
@@ -1,27 +0,0 @@
-/*
- * This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this
- * file, You can obtain one at http://mozilla.org/MPL/2.0/.
- */
-
-/*
- * Copyright (c) 2014, Joyent, Inc.
- */
-
-var bunyan = require('bunyan');
-var helper = require('./helper.js');
-
-
-
-///--- Globals
-
-//TODO: Make this an actual test...
-var test = helper.test;
-
-
-
-///--- Tests
-
-test('stub', function (t) {
-        t.done();
-});
